widget-src/hooks/useInitTheme.ts

```diff
@@ -46,6 +51,20 @@ export const useInitTheme = ({
       borderWidthsSliceItems = currentVariants[BoxPropertyName.BorderWidth];
     }
 
+    const localPaintStyles = figma.getLocalPaintStyles();
+    const existingColors = getColorStateFromPaintStyles(localPaintStyles);
+    if (existingColors.length > 0) {
+      colorSliceItems = existingColors;
+    } else {
+      colorSliceItems = defaultColorSliceItems.map(({ id, name, rgba }) => {
```

Comment:
Maybe we could add all the default colors of Morfeo even if the user has his own color library. Because Morfeo will probably have the same behavior

I think @mauroerta can help us on this.

Reply:
Looks like there's room for a separate discussion. Let's take care of that separately on issue #18

Reply:
Yep, it makes sense. There are no problem if we discuss about this later